cmake/create-symlinks.cmake

```diff
@@ -5,7 +5,7 @@ list(APPEND LLVM_BINARIES clang llvm-link)
 
 # Link directory would differ for cases when we have DESTDIR from CPack or simple installation
 if (DEFINED ENV{DESTDIR})
-    set(LINK_DESTDIR $ENV{DESTDIR}/usr/bin)
+    set(LINK_DESTDIR $ENV{DESTDIR}/var/empty/bin)
```

Comment:
AFAIR the point of these symlinks was having zkllvm binaries in PATH. In particular, CircuitCompile.cmake [relies](https://github.com/NilFoundation/zkLLVM/blob/eb864384bf902c6a2f3a60965bc2a3d6f3338a8a/cmake/CircuitCompile.cmake#L71) on this presence. If we can't make Nix work with `/usr/bin` directory, then we have to rewrite CircuitCompile file, and the symlinks will become unnecessary. Or I would suggest just set `dontInstall = true;` for now.

Reply:
I have re-written it as set(LINK_DESTDIR $ENV{DESTDIR}/usr/bin)